tests/testthat/test-api.R

```diff
@@ -1184,7 +1184,8 @@ test_that("model calibrate can be queued and result returned", {
   expect_equal(status_response$data$status, scalar("COMPLETE"))
   expect_true(status_response$data$success)
   expect_equal(status_response$data$queue, scalar(0))
-  expect_equal(status_response$data$progress, list())
+  expect_match(status_response$data$progress[[1]],
+               "Generating report - [\\d.m\\s]+s elapsed", perl = TRUE)
```

Comment:
surprised you need `perl` here but that's fine too (it probably should be the default...)